code/src/components/NotFound/NotFound.css

```diff
@@ -0,0 +1,16 @@
+.notFound  {
```

Comment:
Add a padding to body? Or perhaps put the max-width of the box to only 80%? Would look good for mobile we think. Now, on mobile, the box takes up all the space.